### PR TITLE
Fix: update README.md with screenshots

### PR DESCRIPTION
**Pull request by mwithi**
Fix screenshots

### CODE REVIEW

**Review by dbmalkovsky — APPROVED** [review]
LGTM as long as "OH 2.0" isn't going to confuse anyone on the Swagger API Documentation image page.

**Comment by mwithi** [comment]
> LGTM as long as "OH 2.0" isn't going to confuse anyone on the Swagger API Documentation image page.

Till now it has actually passed quite ignored... I would keep it just as reminder we are pointing to a new API layer for future applications, otherwise we can just rename it to "Open Hospital Api Documentation" ?

**Comment by dbmalkovsky** [comment]
> > LGTM as long as "OH 2.0" isn't going to confuse anyone on the Swagger API Documentation image page.
> 
> Till now it has actually passed quite ignored... I would keep it just as reminder we are pointing to a new API layer for future applications, otherwise we can just rename it to "Open Hospital Api Documentation" ?

You are the boss 😏 Do what you think is best.

**Comment by mwithi** [comment]
> You are the boss 😏 Do what you think is best.

I was already doing :) found also that version and license need to be aligned with the github project

File: README.md [hunk 1]
```diff
@@ -60,28 +60,28 @@ For the moment, to build this project you should
     - URL patients: http://localhost:8080/patients
     - URL swagger: http://localhost:8080/swagger-ui.html
 
-You can see Swagger Api Documentation at: http://localhost:8080/swagger-ui/
+You can see Swagger Api Documentation at: http://localhost:8080/swagger-ui/index.html
```
Comment (dbmalkovsky) [inline README.md:63]:
```suggestion
You can see Swagger API Documentation at: http://localhost:8080/swagger-ui/index.html
```

typically Api is rendered in all capital letters:  API

File: src/main/java/org/isf/config/OpenAPIConfiguration.java [hunk 2]
```diff
@@ -42,26 +42,22 @@
...
-								     .info(new Info().title("OH 2.0 Api Documentation")
-								     .description("OH 2.0 Api Documentation")
-								     .version("1.0").contact(new Contact().name("ApiInfo.DEFAULT_CONTACT"))
-								     .license(new License().name("Apache 2.0").url("https://www.apache.org/licenses/LICENSE-2.0")))
-								     .servers(List.of(new Server().url("http://localhost:8080")))
-								     .components(new Components().schemas(Map.of(
-								    		 "LocalDate", new DateSchema().name("LocalDate").type("string").format(null),
-								    		 "LocalDateTime", new DateTimeSchema().name("LocalDateTime").type("string").format(null))));
+						new SecurityRequirement().addList("bearerAuth"))
+						.info(new Info().title("Open Hospital Api Documentation")
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/config/OpenAPIConfiguration.java:53]:
```suggestion
						.info(new Info().title("Open Hospital API Documentation")
```

File: src/main/java/org/isf/config/OpenAPIConfiguration.java [hunk 3]
```diff
@@ -42,26 +42,22 @@
...
+										.description("Open Hospital Api Documentation")
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/config/OpenAPIConfiguration.java:54]:
```suggestion
										.description("Open Hospital API Documentation")
```

**Review by dbmalkovsky — APPROVED** [review]
LTGM with minor changes

**Pull request merged**